_sass/layout.scss

```
@@ -45,6 +46,7 @@
 	grid-template-columns: repeat(12, 1fr);
 	grid-template-rows: 115px 876px auto auto;
 	grid-column-gap: 30px;
+	overflow-x: hidden;  // changed here
```

When I'm going to read the code later the common "changed here" won't tell me anything :)

```suggestion
	overflow-x: hidden;
```